src/components/footer/footer.tsx

```diff
@@ -6,11 +8,15 @@ export default function Footer() {
           <h2 className="w-full text-2xl font-extrabold text-accent">Trakr</h2>
         </div>
         <div className="flex w-full gap-6 md:gap-12">
-          <h4 className="cursor-pointer text-sm underline-offset-2 hover:underline">
+          <h4  className="cursor-pointer text-sm underline-offset-2 hover:underline">
+          
             Terms of use
+           
           </h4>
           <h4 className="cursor-pointer text-sm underline-offset-2 hover:underline">
+          <Link href="/privacy-policy">
```

Comment:
Remove the h4 tag, we don't need that
But keep the tailwind classes from it. Use those classes on the Link tag